native/themes/markdown-dark-styles.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import { StyleSheet } from 'react-native';
```

Comment:
I don't think we need to have these in separate files. Most of the styles are identical.

Let's figure out a way to have just a single stylesheet that behaves like `styleSelector`. Instead of having two `autolink` styles, you'll just specify one like this:

```js
  autolink: {
    color: 'link',
    textDecorationLine: 'underline',
  },
```

We won't be able to use `styleSelector` in this way exactly, since it decides whether the content is dark based on app-wide dark mode, rather than based on the color of the chat bubble. But we should be able to reuse `stylesFromColors` from `native/themes/colors.js` for it.

Reply:
I'm aware that they are almost the same, just didn't know exactly how to handle this, but I think your idea will work